CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.13.2]
```

Comment:
There are a bunch of tf changes being worked on right now.  Let's remove this line and bundle it with a larger deploy.